src/index.ts

```diff
@@ -0,0 +1 @@
+export { PJV } from "./PJV";
```

Comment:
I think types like `SpecName` will be useful for folks:

```suggestion
export { PJV } from "./PJV";
export type { SpecName, SpecType } from "./PJV.types"; 
```

_(or `from "./types"` per the comment in that file)_